docker-for-windows/release-notes.md

```diff
@@ -13,10 +13,26 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
+## Docker Desktop 3.5.1
+2021-06-24
+
+[Download](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){: .button .primary-btn}
+
+### New
+
+**Dev Environments Preview**: Dev Environments enable you to seamlessly collaborate with your team members without moving between Git branches to get your code onto your team members' machines. When using Dev Environments, you can share your in-progress work with your team members in just one click, and without having to deal with any merge conflicts. For more information and for instructions on how to use Dev Environments, see [Development Environments Preview](../desktop/dev-environments.md).
+
+**Compose V2 beta**: Docker Desktop now includes the beta version of Compose V2, which supports the `compose` command as part of the Docker CLI. For more information, see [Compose V2 beta](../compose/cli-command.md).
+
+### Bug fixes and minor changes
+- Fixed a bug where users could not install Docker Desktop when the temp folder contained dots. Fixes [docker/for-win#11514](https://github.com/docker/for-win/issues/11514)
```

Comment:
Should it be `the path to the temp folder …`?

Reply:
indeed!